lib/IDETool/CompletionInstance.cpp

```diff
@@ -794,3 +794,70 @@ void swift::ide::CompletionInstance::conformingMethodList(
             Callback);
       });
 }
+
+void swift::ide::CompletionInstance::cursorInfo(
```

Comment:
Little weird it's a part of `CompletionInstance` 😅.

Reply:
Yes, we should rename `CompletionInstance`. But I didn’t know how else to name it.